src/Aspire.Hosting/Publishing/ManifestPublishingContext.cs

```diff
@@ -105,12 +107,37 @@ public async Task WriteContainerAsync(ContainerResource container)
             }
         }
 
-        // Write volume details
+        // Write volume & bind mount details
         if (container.TryGetAnnotationsOfType<ContainerMountAnnotation>(out var mounts))
```

Comment:
(nit) it might make sense to split this out into a private method to make the code more readable.